source/my_custom_package/my_custom_package/test.py

```diff
@@ -13,8 +11,7 @@ def __init__(self, node_name: str, *args, **kwargs):
         self._param_x = sr.Parameter("target_x", 100.0, sr.ParameterType.DOUBLE)
         self._param_y = sr.Parameter("target_y", 100.0, sr.ParameterType.DOUBLE)
         self._param_z = sr.Parameter("target_z", 100.0, sr.ParameterType.DOUBLE)
-        self._param_orientation = sr.Parameter("target_orientation", "[0.0, 0.0, 0.0, 1.0]", sr.ParameterType.STRING)¨
-        self.target_position.set_orientation([1.0, 0.0, 0.0, 0.0])  # Orientation par défaut
+        self._param_orientation = sr.Parameter("target_orientation", [1.0, 0.0, 0.0, 0.0], sr.ParameterType.DOUBLE_ARRAY)
```

Comment:
It's easier to use a double array parameter type than a string. You could also use a double array type for the xyz position with 3 elements, but thats up to you